tests/environment.py

```diff
@@ -84,6 +85,17 @@ def wait_for_log(self, regex: str, start_line: int, timeout: float) -> bool:
         '''Poll the radio's logfile until it contains "regex" or times out.'''
         raise NotImplementedError("wait_for_log is not implemented in abstract class Radio")
 
+    def get_current_channel(self) -> (int, int, int):
+        '''Get the current channel information.
+
+        Returns
+        -------
+        (channel: int, bw: int, center_channel: int)
```

Comment:
Suggestion: You could use a named tuple, it's more self-documenting that way (nitpick)